odrive_state_msgs/srv/SetAxisState.srv

```diff
@@ -0,0 +1,3 @@
+string joint_name
```

Comment:
Add consts for valid axis_states?

Reply:
Similar for other msgs

Reply:
idk. I guess it just means the values are written somewhere convenient. Not a big deal

Reply:
done